opentracing/context.go

```diff
@@ -0,0 +1,35 @@
+package opentracing
+
+// A ContextID is the smallest amount of state needed to describe a span's
+// context within a larger [potentially distributed] trace. The ContextID is
+// not intended to encode the span's operation name, timing, or log data, but
+// merely any unique identifiers (etc) needed to contextualize it within a
+// larger trace tree.
+//
+// ContextIDs are sufficient to propagate the, well, *context* of a particular
+// trace from process to process.
+type ContextID interface {
```

Comment:
Note that this ContextID does *not* specify that the implementations must use a trace_id and span_id, etc, etc... that is a choice the tracing impl can make on its own.